src/pvrclient-nextpvr.h

```diff
@@ -28,7 +28,7 @@
 #include "buffers/RollingFile.h"
 #include "buffers/TimeshiftBuffer.h"
 #include "buffers/TranscodedBuffer.h"
-#include "p8-platform/threads/threads.h"
+#include "kodi/tools/Thread.h"
```

Comment:
No need to include this. Instead include <atomic> and <thread>

Reply:
Thanks I missed removing that thanks.  atomic and thread are already in the genreal Buffer class.

Reply:
Edit, apologies. The atomic and thread did not appear in my original post.